src/Microsoft.SqlTools.ServiceLayer/Scripting/ScriptingService.cs

```diff
@@ -61,7 +63,26 @@ internal static ConnectionService ConnectionServiceInstance
             {
                 connectionService = value;
             }
-        }     
+        }
+
+        /// <summary>
+        /// Internal for testing purposes only
+        /// </summary>
+        internal static LanguageService LanguageServiceInstance
```

Comment:
Minor: it'd be nice if we took opportunities like this to move the scripting service over to the MEF extension-based service definition. That way you never need this static instance reference - instead the test just passes in a service provider that has the mock services as needed. It's pretty quick to change this over and helps get us on 1 standard going forward.